system/lib/libc/musl/arch/generic/bits/errno.h

```diff
@@ -0,0 +1,134 @@
+#define EPERM            1
```

Comment:
It seems we have also errno.h in musl/arch/emscripten/bits? Do we not remove the emscripten version? The same for other added files, such as fcntl.h, fenv.h, shm.h, and termios.h.

Reply:
No, the idea is that we have all the generic headers in `arch/generic`, and then we override just some of them with headers in `arch/emscirpten`.   This is how the musl header layout works.   At install time any emscripten-specific headers will overwrite the generic one.

Reply:
Thanks for the explanation!